commands/moderation/unban.js

```diff
@@ -0,0 +1,61 @@
+const Discord = require('discord.js');
+var dateFormat = require('dateformat');
+
+module.exports = {
+    name: "unban",
+    description: "Unban a user",
+    
+    async execute(msg, args, con) {
+        if (!msg.member.roles.cache.some(
+            role => role.name === "Admin")) {
+                return msg.reply("You must be an Admin to use this command.");
+        }else{
+            let channel = msg.guild.channels.cache.find(channel => channel.name === 'audit-logs')
+
+            const toUnban = args[0];
+
+            if(!toUnban) {
+                return msg.reply("Please include a user ID to unban.")
+            }
+
+            if(msg.client.users.fetch(toUnban) == msg.author) {
+                return msg.reply("You can't unban yourself!")
+            }
```

Comment:
If the user is banned, they won't be able to use the bot, right? If so, this might not be necessary.

Reply:
This was for if you tried to unban yourself.